src/content/guides/installing-optional-ingress-controller/index.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Installing an Ingress Controller"
+description = "How to install the nginx-ingress-controller using our Web UI"
+date = "2020-02-04"
+type = "page"
+weight = 50
+tags = ["tutorial"]
++++
+
+
+# Installing an Ingress Controller
+
+An ingress controller helps you expose your services to the outside world.
+
+Not all of our clusters come with an ingress controller pre-installed. We are
+currently in a shift where certain releases have a pre-installed ingress controller
+and certain releases require you to install one yourself.
+
+Eventually all releases will not ship with a built-in ingress controller. That
+allows you full control to choose what and how many ingress controllers you
+want to run on your cluster.
+
```

Comment:
The grammar is weird here. I propose:
_Eventually, all releases will ship without a built-in ingress controller_. That allows you full control to choose _which_ and how many....